README.rst

```diff
@@ -685,16 +685,14 @@ This only applies in command files, which are heuristically detected as files wi
 ``EmailValidator``
 ~~~~~~~~~~~~~~~~~~
 
-Rewrites keyword arguments to their new names: ``whitelist`` to ``allowlist``, and ``domain_whitelist`` to ``domain_allowlist``.
+Rewrites keyword arguments to their new names: ``whitelist`` to ``allowlist``.
```

Comment:
```suggestion
Rewrites the ``whitelist`` keyword argument to its new name ``allowlist``.
```

Reply:
You didn't applied this suggestion before merging, not sure it was intended ?

Reply:
woops!

Reply:
done in 76819494042fc3aaaa01604ed40418f4ce0078b0